### ISSUE TITLE
aria-readonly for checkboxes is not output

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#425]
### Summary

1. Go to https://codepen.io/jaws-test/pen/mdPemVX
2. Navigate to the readonly checkboxes (2nd and 4th)

### Expected result

Output of readonly

### Actual result

No output of readonly

#### JAWS version and build number
2020.2006
 
#### Operating System and version
Windows 10

#### Browser and version:
current versions of Firefox ESR, IE 11 and Chrome

**Comment by menovak** [comment]
testing:

Per testing provided codepen in Windows 10, confirmed that JAWS 2020.2003 did not announce the 'readonly' attribute per the 2nd and 4th checkboxes in Chrome 84, in Firefox 79 and in IE11.   This seems to be somewhat of a backtrack, as JAWS 2018.1807 did announce the 'readonly' attribute per the 2nd and 4th checkboxes in Chrome 84 and in IE11.   

In review, the 'readonly' attribute appears to be in the accessibility tree for Chrome 84, but in Firefox 79, only the 4th checkbox has a readonly state that appears in the accessibility tree.

NVDA 2020.2 announces the 'readonly' attribute per the 2nd and 4th checkboxes in Chrome 84 and in IE11, but for Firefox 79 it only announced the 'readonly' attribute on the 4th checkbox, which seems to follow what is exposed in the accessibility tree in Firefox.   

If we change the code for the 2nd checkbox from:
<input type=checkbox aria-label=2 aria-readonly=true>

to:

<input type="checkbox" aria-label="2" aria-readonly="true" role="checkbox">

We can improve how NVDA announces the 'readonly' attribute for the 2nd checkbox in Firefox, by adding the role="checkbox", Firefox now exposed the readonly' attribute on the 2nd checkbox.  This change, however, did not improve or change how JAWS 2020 announces the attribute as shown in the test results below.

results:
1 - appears JAWS 2020 has taken a step back from what JAWS 2018 announced in Chrome and IE11
2 - appears even if the 'readonly' attribute is exposed in the accessibility tree in Firefox, that JAWS (either version) does not announce the attribute properly




--- speech output results below from TAB to each checkbox original codepen ---

//NVDA 2020.2 & Chrome 84
1  check box  not checked  
2  check box  not checked  read only  
3  check box  checked  
4  check box  checked  read only  

// NVDA 2020.2 & IE11
1  check box  not checked  
2  check box  not checked  read only  
x  3  check box  checked  
x  4  check box  checked  read only 

// NVDA 2020.2 & FireFox 79
1  check box  not checked  
2  check box  not checked  
3  check box  checked  
4  check box  checked  read only

-------------------------------------------

// JAWS 2020.2003 & Chrome 84
1 check box not checked 
To check press Spacebar.
2 check box not checked 
To check press Spacebar.
3 check box checked 
To clear checkmark press Spacebar.
4 check box checked 
To clear checkmark press Spacebar.

// JAWS 2020.2003 & FireFox 79
1 check box not checked 
To check press Spacebar.
2 check box not checked 
To check press Spacebar.
3 check box checked 
To clear checkmark press Spacebar.
4 check box checked 
To clear checkmark press Spacebar.

// JAWS 2020.2003 & IE11 
1 check box not checked 
To check press Spacebar.
2 check box not checked 
To check press Spacebar.
3 check box checked 
To clear checkmark press Spacebar.
4 check box checked 
To clear checkmark press Spacebar.

  
---------------------------------------------------------


// JAWS 2018.1807 & Chrome 84
1 check box not checked 
2 check box not checked Read Only 
3 check box checked 
4 check box checked Read Only

// JAWS 2018.1807 & Firefox 79
1 check box not checked 
To check press Spacebar.
2 check box not checked 
To check press Spacebar.
3 check box checked 
To clear checkmark press Spacebar.
4 check box checked 

// JAWS 2018.1807 & IE11
4 check box not checked 
4 check box not checked Read Only 
4 check box checked 
4 check box checked Read Only


--- speech output results below from TAB to each checkbox in altered codepen, added role="checkbox" to the 2nd checkbox ---

nvda 2020.2 & Chrome 84
1  check box  not checked  
2  check box  not checked  read only  
3  check box  checked  
4  check box  checked  read only  

nvda 2020.2 & Firefox 79
1  check box  not checked  
2  check box  not checked  read only  
3  check box  checked  
4  check box  checked  read only  

nvda 2020.2 & ie11
1  check box  not checked  
2  check box  not checked  read only  
x  3  check box  checked  
x  4  check box  checked  read only  



jaws 2020 & chrome  84
1 check box not checked 
To check press Spacebar.
2 check box not checked 
To check press Spacebar.
3 check box checked 
To clear checkmark press Spacebar.
4 check box checked 
To clear checkmark press Spacebar.


jaws 2020 & Firefox 79
note 'readonly' exposed in a11y tree on checkbox 2 and 4 but still not announced....

1 check box not checked 
To check press Spacebar.
2 check box not checked 
To check press Spacebar.
3 check box checked 
To clear checkmark press Spacebar.
4 check box checked 
To clear checkmark press Spacebar.

jaws 2020 & ie11 
1 check box not checked 
To check press Spacebar.
2 check box not checked 
To check press Spacebar.
3 check box checked 
To clear checkmark press Spacebar.
4 check box checked 
To clear checkmark press Spacebar.


----------

jaws 2018  & Chrome 84
4 check box checked Read Only 
3 check box checked 
2 check box not checked Read Only 
1 check box not checked

jaws 2018  & Firefox 79
1 check box not checked 
To check press Spacebar.
2 check box not checked 
To check press Spacebar.
3 check box checked 
To clear checkmark press Spacebar.
4 check box checked 
To clear checkmark press Spacebar.

jaws 2018 & ie11
1 check box not checked 
2 check box not checked Read Only 
3 check box checked 
4 check box checked Read Only

**Comment by JAWS-test** [comment]
See: https://github.com/w3c/aria/issues/1309
